### DISCUSSION TITLE
Massive disabling of Fuzzy matching

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5626] [Ideas] [answered]
I need to turn off Fuzzy matching in 800 files (not all in the project), how to do it as soon as possible? I read the documentation for "Management commands" in the section [command#install-addon](https://docs.weblate.org/en/latest/admin/management.html?highlight=command#install-addon), there is only rudimentary information.

If I understand correctly, at present the addon settings in the "base component" only apply to new components. There is no function to change addon settings in ui for more components.

**Comment by nijel** [comment, marked as answer]
Use the API to adjust addons configuration: https://docs.weblate.org/en/latest/api.html#addons
